core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/tail.kt

```diff
@@ -1,9 +1,12 @@
 package org.jetbrains.kotlinx.dataframe.api
 
 import org.jetbrains.kotlinx.dataframe.DataFrame
+import org.jetbrains.kotlinx.dataframe.util.TAIL
+import org.jetbrains.kotlinx.dataframe.util.TAIL_REPLACE
 
 // region DataFrame
 
+@Deprecated(TAIL, ReplaceWith(TAIL_REPLACE), DeprecationLevel.ERROR)
```

Comment:
I would keep `tail` as shortcut for `takeLast(5)`, since we also have `head` as a shortcut for `take(5)`. It's easy to get a quick glance at the beginning or end of a dataframe